packages/realm-server/server.ts

```diff
@@ -656,6 +658,13 @@ export class RealmServer {
     throw new Error(`Can not determine the matrix registration secret`);
   }
 
+  private handleStripeWebhook = async (ctxt: Koa.Context, _next: Koa.Next) => {
```

Comment:
is there a way to auth the request so not just anyone can call this endpoint?

Reply:
nvm, i just noticed there is a `STRIPE_WEBHOOK_SECRET` in the handleStripeWebhook

Reply:
Yeah, that will allow us to secure this endpoint. I'll have a PR for adding that ENV var soon